agent/bench-scripts/tests/pbench-fio/test-04.txt

```diff
@@ -9,7 +9,7 @@ direct=1
 sync=0
 time_based=1
 clocksource=gettimeofday
-ramp_time=5
+ramp_time=61
 write_bw_log=fio
 write_iops_log=fio
 write_lat_log=fio
```

Comment:
_[This question is probably aimed more at Peter....]_
The job file for `test-04` provides `log_hist_msec=10000`, which for a 30 second run (especially if there is a 61 second ramp up) seems like it should produce at least 2 and possibly 10 log history records...but the test is complaining that it finds none.  Is this good and appropriate, or is there something wrong with the test or the configuration?  If it's good, then should we be omitting the `log_hist_msec` from the job file, so that we don't get the warnings?

Reply:
Tracked in #2641.